doc/source/dev/governance/governance.rst

```diff
@@ -378,6 +378,22 @@ A list of current Institutional Partners is maintained at the page
 :ref:`governance-people`.
 
 
+Commit Rights in the NumPy Code Base
+====================================
+
+The NumPy Steering Council can grant and revoke commit rights to the NumPy code
+repository, currently hosted at `github`_.
+
+The policy for granting commit priveleges is based on trust, with the
+understanding that they are needed for many of the mundane tasks required
+to maintain NumPy. New people should be conservative in exercising their
```

Comment:
Instead of *new people should* I would choose something along the lines of

> Upon receiving this priviledge, it is expected from the members to use it conservatively and mostly use it to triage issues, add labels, and rather merge only the safe pull requests such as documentation.

Reply:
revised